packages/frontend/editor-ui/src/router.ts

```diff
@@ -838,6 +838,7 @@ router.beforeEach(async (to: RouteLocationNormalized, from, next) => {
 			console.error(failure);
 		}
 	}
+	return next();
```

Comment:
This seems a bit risky, to force next even in case of error, whereas this was not the case before if I understand correctly 🤔